### PR TITLE
[README] Add Section for Community Projects

### PR DESCRIPTION
**Pull request by JacobEverly**
Adding section to highlight community projects building on top of the HyperSDK

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -988,6 +988,15 @@ _This is a collection of posts from the community about the `hypersdk` and how t
 * [Launching Custom Tokens With HyperSDK By Avalanche](https://pythontony.hashnode.dev/launching-custom-tokens-with-hypersdk-by-avalanche)
 * [Avalanche VMs deep-dive #1: HyperSDK/tokenvm](https://ashavax.hashnode.dev/avalanche-vms-deep-dive-1-hypersdktokenvm)
 
+## HyperSDK Projects 👀
+_This is a gallery of projects currently building on top of the HyperSDK
```
Comment (hexfusion) [inline README.md:992]:
```suggestion
This is a gallery of projects currently building on top of the HyperSDK
```

File: README.md [hunk 2]
```diff
@@ -988,6 +988,15 @@ _This is a collection of posts from the community about the `hypersdk` and how t
...
+
+* [NodeKit: Decentralizing The L2 Sequunecer on a Subnet](https://nodekit.gitbook.io/nodekit-documentation/)
```
Comment (hexfusion) [inline README.md:994]:
```suggestion
* [NodeKit: Decentralizing The L2 Sequencer on a Subnet](https://nodekit.gitbook.io/nodekit-documentation/)
```

File: README.md [hunk 3]
```diff
@@ -988,6 +988,15 @@ _This is a collection of posts from the community about the `hypersdk` and how t
...
+* [OracleVM: Providing OffChain Data to the Avalanche Ecosystem](https://github.com/bianyuanop/oraclevm)
+* [ShuttleVM: Trusted Execution Environment for Wasm Programs to run distributed workloads on a subnet]
```
Comment (hexfusion) [inline README.md:996]: link? I think the requirement to list a project is to have the project open source or a minimum link to something.

**Review by hexfusion — COMMENTED** [review]
Good idea, I would like to see project maintainers adding these links vs you. :)

File: README.md [hunk 4]
```diff
@@ -988,6 +988,15 @@ _This is a collection of posts from the community about the `hypersdk` and how t
...
 
+## HyperSDK Projects 👀
+This is a gallery of community projects currently building on top of the HyperSDK
+
+* [NodeKit: Decentralizing The L2 Sequencer on a Subnet](https://nodekit.gitbook.io/nodekit-documentation/)
+* [OracleVM: Providing OffChain Data to the Avalanche Ecosystem](https://github.com/bianyuanop/oraclevm)
+* [ShuttleVM: Trusted Execution Environment for Wasm Programs to run distributed workloads on a subnet]
+
+
+
```
Comment (hexfusion) [inline README.md:999]: minor: single newline between sections

File: README.md [hunk 5]
```diff
@@ -988,6 +988,15 @@ _This is a collection of posts from the community about the `hypersdk` and how t
 * [Launching Custom Tokens With HyperSDK By Avalanche](https://pythontony.hashnode.dev/launching-custom-tokens-with-hypersdk-by-avalanche)
 * [Avalanche VMs deep-dive #1: HyperSDK/tokenvm](https://ashavax.hashnode.dev/avalanche-vms-deep-dive-1-hypersdktokenvm)
 
+## HyperSDK Projects 👀
+This is a gallery of community projects currently building on top of the HyperSDK
+
+* [NodeKit: Decentralizing The L2 Sequencer on a Subnet]()
+* [OracleVM: Providing OffChain Data to the Avalanche Ecosystem]()
+* [ShuttleVM: Trusted Execution Environment for Wasm Programs to run distributed workloads on a subnet]
```
Comment (JacobEverly) [inline README.md:996]:
```suggestion
* [ShuttleVM: Trusted Execution Environment for Wasm Programs to run distributed workloads on a subnet]()
```

File: README.md (hunk 3 above)
Comment (patrick-ogrady) [reply README.md:996]: I think a link is a good minimum requirement (even if to a website instead of the code at first)

File: README.md [hunk 6]
```diff
@@ -988,6 +988,15 @@ _This is a collection of posts from the community about the `hypersdk` and how t
 * [Launching Custom Tokens With HyperSDK By Avalanche](https://pythontony.hashnode.dev/launching-custom-tokens-with-hypersdk-by-avalanche)
 * [Avalanche VMs deep-dive #1: HyperSDK/tokenvm](https://ashavax.hashnode.dev/avalanche-vms-deep-dive-1-hypersdktokenvm)
 
+## HyperSDK Projects 👀
+This is a gallery of community projects currently building on top of the HyperSDK
```
Comment (patrick-ogrady) [inline README.md:992]: nit: add a period, use italics like `Community Posts`

Comment (JacobEverly) [inline README.md:992]:
```suggestion
_This is a gallery of community projects currently building on top of the HyperSDK._
```

File: README.md [hunk 7]
```diff
@@ -988,6 +988,15 @@ _This is a collection of posts from the community about the `hypersdk` and how t
 * [Launching Custom Tokens With HyperSDK By Avalanche](https://pythontony.hashnode.dev/launching-custom-tokens-with-hypersdk-by-avalanche)
 * [Avalanche VMs deep-dive #1: HyperSDK/tokenvm](https://ashavax.hashnode.dev/avalanche-vms-deep-dive-1-hypersdktokenvm)
 
+## HyperSDK Projects 👀
+_This is a gallery of community projects currently building on top of the HyperSDK._
+
+* [NodeKit: Decentralizing The L2 Sequencer on a Subnet]()
```
Comment (patrick-ogrady) [inline README.md:994]: https://github.com/AnomalyFi/nodekit-seq

File: README.md [hunk 8]
```diff
@@ -988,6 +988,15 @@ _This is a collection of posts from the community about the `hypersdk` and how t
...
+* [OracleVM: Providing OffChain Data to the Avalanche Ecosystem]()
```
Comment (patrick-ogrady) [inline README.md:995]: https://github.com/bianyuanop/oraclevm

File: README.md [hunk 9]
```diff
@@ -988,6 +988,15 @@ _This is a collection of posts from the community about the `hypersdk` and how t
...
+* [ShuttleVM: Trusted Execution Environment for Wasm Programs to run distributed workloads on a subnet]()
```
Comment (patrick-ogrady) [inline README.md:996]: https://github.com/Shuttle-VM/Shuttlecock

**Pull request merged**